### PR TITLE
[Cookbook][Security] proofread comments in voter article

### PR DESCRIPTION
**Pull request by xabbuh**
| Q | A |
| --- | --- |
| Doc fix? | yes |
| New docs? | no |
| Applies to | all |
| Fixed tickets |  |

### CODE REVIEW

File: cookbook/security/voters.rst [hunk 1]
```diff
@@ -134,9 +134,9 @@ edit a particular object. Here's an example implementation::
                 return VoterInterface::ACCESS_DENIED;
             }
 
-            // double-check that the User object is the expected entity.
-            // It always will be, unless there is some misconfiguration of the
-            // security system.
+            // double-check that the User object is the expected entity
+            // it always will be unless there is some misconfiguration of the
```
Comment (OskarStark) [inline cookbook/security/voters.rst]: the whole sentence sounds weird IMO

**Comment by OskarStark** [comment]
:+1:

File: cookbook/security/voters.rst (hunk 1 above)
Comment (javiereguiluz) [inline cookbook/security/voters.rst]: I agree. What about removing this part? `// it always will be unless there is some misconfiguration of the security system`

Comment (OskarStark) [inline cookbook/security/voters.rst]: i would say that the first part is enough: ```// double-check that the User object is the expected entity.```

Comment (WouterJ) [inline cookbook/security/voters.rst]: I believe @weaverryan liked to have the second sentence, indicating that this is a very rare exception.

Comment (OskarStark) [inline cookbook/security/voters.rst]: maybe like this:

```
    // double-check that the User object is the expected entity.
    // this is a very rare case and only occurs, if you missconfigured your
    // security system.
```

Comment (xabbuh) [inline cookbook/security/voters.rst]: What about:

> double-check that the User object is the expected entity (this only happens when you did not configure the security system properly)

Comment (OskarStark) [inline cookbook/security/voters.rst]: sounds good to me

Comment (xabbuh) [inline cookbook/security/voters.rst]: updated here

**Comment by WouterJ** [comment]
:+1:

**Pull request merged**